kubernetes/samples/scripts/create-rcu-schema/common/createRepository.sh

```diff
@@ -7,7 +7,9 @@
 echo "Check if the DB Service is ready to accept request "
 connectString=${1:-oracle-db.default.svc.cluster.local:1521/devpdb.k8s}
 schemaPrefix=${2:-domain1}
-echo "DB Connection URL [$connectString] and schemaPrefix is [${schemaPrefix}]"
+rcuType=${3:-fmw}
```

Comment:
you are introducing a new parameter right? seems like there should be a matching documentation update in this PR too please

Reply:
This input parameter (rcuType) is determined automatically in the parent script  create-rcu-schema.sh as described in the description of PR. Here we are not adding new input parameter but determine the rcuType form the rcu image.

Reply:
ok